etrago/appl.py

```diff
@@ -327,8 +327,8 @@ def run_etrago(args, json_path):
         eTraGo result network based on `PyPSA network
         <https://www.pypsa.org/doc/components.html#network>`_
     """
-    etrago = Etrago(args, json_path)
-
+    etrago = Etrago(args, json_path= None)
```

Comment:
Do you want to merge this? It looks more like a temporal adjustment...

Reply:
The "= None" was deleted.